test/nuxeo-favorites-toggle-button.html

```diff
@@ -39,7 +39,7 @@
     </template>
   </test-fixture>
 
-  <test-fixture id="favorite">
+  <test-fixture id="fav">
```

Comment:
Does this make any difference?

Reply:
It does. Without it, the test fails. Not sure why, but there seems to be some sort of conflict...